### PR TITLE
Component rendering tests

### PR DESCRIPTION
**Pull request by HeathWil**

### CODE REVIEW

File: packages/tempest/styleMock.ts [hunk 1]
```diff
@@ -0,0 +1 @@
+export default {}
```
Comment (DuncanWalter) [inline packages/tempest/styleMock.ts]: Let's add a comment explaining why this needs to be here- it's a bit confusing at first glance if you don't know about the jest configuration problems

File: packages/tempest/src/TextTest.test.tsx [hunk 2]
```diff
@@ -0,0 +1,20 @@
+import {cleanup} from 'react-testing-library'
+import {Text} from './Text'
+import * as React from 'react'
+import * as enzyme from 'enzyme'
+import * as Adapter from 'enzyme-adapter-react-16';
```
Comment (DuncanWalter) [inline packages/tempest/src/TextTest.test.tsx]: It looks like prettier isn't running here- was there an issue with it?

**Comment by DuncanWalter** [comment]
Ok, how brave are you feeling? It looks like there is an `esModuleInterop` setting that needs to be turned to `true` in our `tsconfig.json`. That will make the `import * as foo` illegal and make the syntax we were originally using work properly (I hope). I could go in and see if that works or you can

**Comment by HeathWil** [comment]
I'm down for trying to implement all your changes! I can work with you if there are any issues. This week is a little busy for me, but I'll do my best to squeeze in as much as I can. Also I can try to change the esModuleInterop setting

File: packages/tempest/src/TextTest.test.tsx (hunk 2 above)
Comment (HeathWil) [reply packages/tempest/src/TextTest.test.tsx]: To be honest I'm not sure - I was very focused on trying to get the functionality of the test first. I can try to debug why its not working when I get a chance

File: packages/tempest/styleMock.ts (hunk 1 above)
Comment (HeathWil) [reply packages/tempest/styleMock.ts]: will do

**Comment by HeathWil** [comment]
It still looks like prettier isn't working (at least to me) and I'm not sure why

File: packages/tempest/src/Text.tsx [hunk 3]
```diff
@@ -1,4 +1,4 @@
-import React from 'react'
+import * as React from 'react'
```
Comment (DuncanWalter) [inline packages/tempest/src/Text.tsx]: Looks like some of these need to be switched back to the import `React from 'react'` syntax?

File: packages/tempest/src/Checkbox.tsx [hunk 4]
```diff
@@ -1,4 +1,4 @@
-import React, { Component } from 'react'
+import * as React from 'react'
```
Comment (DuncanWalter) [inline packages/tempest/src/Checkbox.tsx]: Same as Text

File: packages/tempest/src/Card.tsx [hunk 5]
```diff
@@ -1,4 +1,5 @@
-import React, { ReactChild, Component, ReactNode } from 'react'
+import * as React from 'react'
```
Comment (DuncanWalter) [inline packages/tempest/src/Card.tsx]: Same as Text

File: packages/tempest/src/Button.tsx [hunk 6]
```diff
@@ -1,4 +1,4 @@
-import React from 'react'
+import * as React from 'react'
```
Comment (DuncanWalter) [inline packages/tempest/src/Button.tsx]: Same as Text

**Comment by DuncanWalter** [comment]
:shipit:

**Pull request merged**